src/Desktop/ColorContrastAnalyzer/BitmapCollection.cs

```diff
@@ -5,27 +5,27 @@ namespace Axe.Windows.Desktop.ColorContrastAnalyzer
 {
     public class BitmapCollection : ImageCollection
     {
-        private readonly System.Drawing.Bitmap bitmap;
+        private readonly System.Drawing.Bitmap _bitmap;
```

Comment:
I renamed this member from `bitmap` to `_bitmap` since it was so straightforward and allowed the remove of `this.` prefixes.